1811 - Fix multiselect checkboxes and item text align for rtl with searchfiled (#3299)
infor-design/enterprise: docs/CHANGELOG.md

- `[Locale]` Changed incorrect Chinese locale year formats to the correct format as noted by translators. For example `2019年 12月`. ([#3081](https://github.com/infor-design/enterprise/issues/3081))
- `[Locale]` Corrected and added the firstDayofWeek setting for every locale. ([#3060](https://github.com/infor-design/enterprise/issues/3060))
- `[Modal]` Fixed an issue where the returns focus to button after closing was not working. ([#3166](https://github.com/infor-design/enterprise/issues/3166))

> - `[Searchfiled]` Fixed an issue where the multiselect checkboxes and item text was misaligned for rtl. ([#1811](https://github.com/infor-design/enterprise/issues/1811))

Reviewer: ```suggestion
- `[Searchfield]` Fixed an issue where multiselect items' checkboxes and text were misaligned in RTL mode. ([#1811](https://github.com/infor-design/enterprise/issues/1811))
```
